chia/full_node/coin_store.py

```diff
@@ -61,45 +60,48 @@ async def create(cls, db_wrapper: DBWrapper, cache_size: uint32 = uint32(60000))
         self.coin_record_cache = LRUCache(cache_size)
         return self
 
-    async def new_block(self, block: FullBlock, tx_additions: List[Coin], tx_removals: List[bytes32]):
+    async def new_block(
+        self,
+        height: uint32,
+        timestamp: uint64,
+        included_reward_coins: Set[Coin],
```

Comment:
this is the key change. Instead of taking a `FullBlock` object, take the 3 parts of it that we need. height, timestamp and the reward coins